### PR TITLE
exception when request app by name

### PR DESCRIPTION
**Pull request by SiweiWang**

### CODE REVIEW

File: models/dbschema.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ var BuildSchema = new Schema ({
   gitBranch: String,
   gitSHA: String,
   dockerDigest: String,
+  commits: String,
```
Comment (c4po) [inline models/dbschema.js]: what's this for?

Comment (SiweiWang) [inline models/dbschema.js]: track the commit in the build

Comment (c4po) [inline models/dbschema.js]: we already have a gitSHA

Comment (SiweiWang) [inline models/dbschema.js]: How do you track commits easily in that build?  I don't know a easy way to use gitsha the delta commit .

Comment (SiweiWang) [inline models/dbschema.js]: I have taken out the commits attributes for now,  But I think we will need a easy way to track it . Show the delta commits in the build is a very useful feature IMHO

**Comment by c4po** [comment]
can you give an example of "commits" ?
do you mean the commit messages?

**Comment by SiweiWang** [comment]
More or less, a build could container 1 or more delta commit. The commit container the sha for that change , the commit message and commiter. For example 

time 
^   SHA2  Build 2  
|   SHA2  Add featrue B   by Max
|   SHA2   Add feature A   by Siwei
|   SHA1   Build 1

In this case , the commit String (or Object) should be an array containers 2 object representing each commit. So we what are changed for that build. (without changing / refering back to git or Jenkins)

**Comment by c4po** [comment]
how to define and track the delta?

if build #1 is from branch a, and build #2 is from build b, while we build #3 on branch a, should we compute the delta between #1 and #3 or #2 and #3?

**Comment by SiweiWang** [comment]
My thinking is from the time we merge and build . we can use git log to check the commit different of two branches . i.e. git log --graph --pretty=format:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr)%Creset' --abbrev-commit --date=relative master..branch-X

**Comment by SiweiWang** [comment]
or  git log development..qa

Where development and qa are the branch name

**Pull request closed without merging**